bigtable-hbase-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/RowMergerPerf.java

```diff
@@ -100,7 +100,7 @@ private static void rowMergerPerf(ReadRowsResponse response, int cellCountPerRow
       // The adapter is slower than the RowMerger, so decrease the number of rows by a factor of 3
       // so that the test finishes faster. This will be enough of a sample to get an idea about the
       // adapter's performance
-      long rowCount = CUMULATIVE_CELL_COUNT / cellCountPerRow / 4;
+      long rowCount = CUMULATIVE_CELL_COUNT / cellCountPerRow;
```

Comment:
The new change is more performance sensitive to families with a larger number of qualifiers.  The sorting is of values within the qualifiers.  The more qualifiers, the more `(n * log n)` takes.